subsys/net/lib/lwm2m/lwm2m_util.c

```diff
@@ -422,3 +424,54 @@ int lwm2m_ftoa(double *input, char *out, size_t outlen, int8_t dec_limit)
 			/* handle negative val2 when val1 is 0 */
 			(val1 == 0 && val2 < 0) ? "-" : "", (long long)val1, buf);
 }
+
+int lwm2m_path2string(char *buff, size_t buff_size, struct lwm2m_obj_path *input, int level_max)
+{
+	if (!buff || !buff_size || !input) {
+		return -EINVAL;
+	}
+
+	if (input->level == LWM2M_PATH_LEVEL_NONE) {
```

Comment:
should be /?

Reply:
plus end

Reply:
Please elaborate as I don't get the idea. :)

Reply:
Fixed, I hope.